editor/src/components/inspector/common/inspector-end-to-end-tests.spec.browser2.tsx

```diff
@@ -54,6 +55,18 @@ async function getControlValue(controlTestId: string, renderedDOM: RenderResult)
   return control.value
 }
 
+function elementDoesNotExist(renderResult: EditorRenderResult, testId: string) {
```

Comment:
https://testing-library.com/docs/queries/about

there's a cheat sheet table on this page, there's queryBy and queryAllBy which don't throw error on missing elements

Reply:
😱

Reply:
https://github.com/concrete-utopia/utopia/pull/3316/commits/6e4f7a44736a3038158a0779f52335c1d79278fc